.github/workflows/ci.yaml

```diff
@@ -104,6 +104,7 @@ jobs:
       EXPERIMENTAL_FEATURES: 1
       COMPAT: 0
       PYTEST_PAR: 2
+      PYTEST_OPTS: "--timeout=300"
```

Comment:
it is possible also to use the Github action timeout `timeout-minutes: 300` in order to make stay safe at all level

Reply:
No, I want this inside the test, so we get a backtrace...

Reply:
Oh! I tested it and I got what you want!

I introduce in lnprototest also the logger that you can enable by passing to pytest `--log-cli-level=DEBUG`!